cmd/lotus-miner/proving.go

```diff
@@ -321,12 +354,12 @@ var provingDeadlineInfoCmd = &cli.Command{
 		fmt.Printf("Current:                  %t\n\n", di.Index == dlIdx)
 
 		for pIdx, partition := range partitions {
-			sectorCount, err := partition.AllSectors.Count()
+			sectorCount, err := partition.ActiveSectors.Count()
 			if err != nil {
 				return err
 			}
 
-			sectorNumbers, err := partition.AllSectors.All(sectorCount)
+			sectorNumbers, err := partition.ActiveSectors.All(sectorCount)
 			if err != nil {
```

Comment:
Let's add a flag `all` to this function to give the user the option to get info for `AllSectors`

Reply:
maybe AllSectors is useless to the user